src/istgt_iscsi.c

```diff
@@ -7180,6 +7206,39 @@ istgt_iscsi_drop_all_conns(CONN_Ptr conn)
 	return (0);
 }
 
+// istgt_iscsi_get_existing_live_conn returns existing live
```

Comment:
better to put it under macro `REPLICATION`

Reply:
updated